hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/protocolPB/CommonPBHelper.java

```diff
@@ -0,0 +1,135 @@
+/**
```

Comment:
Currently PBHelper is in module hadoop-hdfs-project and we cannot use it for codes under module hadoop-common, so I add CommonPBHelper in this patch.